src/WebJobs.Script.ConsoleHost/App.config

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you name this project (and related projects) WebJobs.Script.CLI? That leads to less confusion with the existing 2 hosts we have, which are about hosting functions, not CLI stuff.